pypeit/tests/test_cooked.py

```diff
@@ -20,6 +20,6 @@ def test_cooked_version():
     v_file = os.path.join(os.getenv('PYPEIT_DEV'), 'Cooked', 'version')
     with open(v_file) as f:
         tmp = f.readlines()
-    assert tmp[-1].strip() == '1.4.4'
+    assert tmp[-1].strip() == '1.4.5'
```

Comment:
I see you uploaded a new Cooked file.  Did you add files to it?
I'm guessing not given what I've seen (i.e. no new tests other
than the `load_image`.  

If no changes, you can skip that step, remove the 1.4.5. file and set this back.

Reply:
Reverted.